uber/tests/models/test_attendee.py

```diff
@@ -354,6 +354,7 @@ class TestBadgeAdjustments:
     @pytest.fixture(autouse=True)
     def mock_attendee_session(self, monkeypatch):
         monkeypatch.setattr(Attendee, 'session', Mock())
+        Attendee.session.get_next_badge_num = Mock(return_value=123)
```

Comment:
@kitsuta FYI, I did the merge and this was the only line that changed.  I think it -might- be incorrect, need to check, it was confusing what line the merge was adding or removing here.